src/test/dart_debug/debug/dart_test.test.ts

```diff
@@ -353,6 +354,39 @@ describe("dart test debugger", () => {
 		}
 	}).timeout(160000); // This test runs lots of tests, and they're quite slow to start up currently.
 
+	it("can rerun only skipped tests", async () => {
+		await openFile(helloWorldTestTreeFile);
+		const config = await startDebugger(helloWorldTestTreeFile);
+		config!.noDebug = true;
+		await waitAllThrowIfTerminates(dc,
+			dc.configurationSequence(),
+			dc.waitForEvent("terminated"),
+			dc.launch(config),
+		);
+
+		// Now re-run only failed tests.
+		await vs.commands.executeCommand("dart.runAllSkippedTestsWithoutDebugging");
+
+		await openFile(helloWorldTestTreeFile);
+		// Get the expected tree and filter it to only skipped tests.
+		// const expectedResults = getExpectedResults().split("\n").filter((l) => l.includes("skip.svg")).join("\n");
```

Comment:
This comment is out of date, and the commented code can be removed. I presume the `expectedResults` variable below is to account for differences when running skipped tests from the normal expected results embedded in the file.